src/app/about/page.tsx

```diff
@@ -1,7 +1,111 @@
 import Image from "next/image";
 import styles from "./About.module.css";
 
+import OfficerCard, { OfficerCardProperties } from "./officer-card";
+
+const officerCardData: OfficerCardProperties[] = [
```

Comment:
[Comment]: This may be a good candidate to have in a separate file, that way it can just be referenced by this page. I'd expect this constant to get a lot messier once we start adding previous officers.

Reply:
I agree. I like to put data like this in separate files. In previous projects, I've stored data in `.ts` files in a separate `data/` directory. We could do that too, or put data in the `src/app/about/` directory. I'd like to know what you think (and if this should maybe be saved for a separate issue).

Reply:
I'm generally inclined to always abstract by usage - in other words, a simple `officers.ts` file (or `officer-data.ts`, if preferred) in the `about` directory is probably the best option for now. I'm not particularly a fan of diving projects up into categories like `/components`, `/forms`, `/types`, etc. I think it forces more abstraction than is often needed (or conceptually present) and grouping by programmatic concern is less valuable than grouping by business concern.